source/theory.rst

```diff
@@ -440,16 +457,27 @@ The Morison Equation assumes that the fluid forces in an oscillating flow on a s
 
 where :math:`v` is the fluid particle velocity due to wave and current, :math:`C_{a}` is the coefficient of added mass, and :math:`\forall` is the displaced volume. 
 
+.. Note:: 
+	Adam:
+	What 'resulting force' can be approdimated by the Morison formulation? Some more information is needed here. My understanding is that it accounts for the undisturbed wave (Froude-Krylov force), viscous drag and added mass. So then should F_exc = F_rad = F_b = F_v = 0 when F_me is used? The below note only states that buoyance is neglected, not all of these components. (My understanding may be wrong of course, but if so I think that it shows some more description is needed.)
+
```

Comment:
@akeeste and @nathanmtom you should coordinate on this comment, since the docs will need to be updated once PR #408 is merged.

Reply:
@kmruehl  I will move this comment to my new PR after looking at the documentation Nathan creates